frontend/app/src/components/status/notifications/Notification.vue

```diff
@@ -96,9 +112,33 @@ const action = async (notification: NotificationData) => {
       <div
         class="mt-1 px-2"
         :style="fontStyle"
-        :class="[$style.message, { [$style.inline]: !popup }]"
+        :class="[css.message, { [css.inline]: !popup }]"
       >
-        <div>{{ notification.message }}</div>
+        <template v-if="isMissingKeyNotification">
+          <i18n
+            v-if="notification.i18nParam"
+            :path="notification.message"
+            :plural="notification.i18nParam.choice"
+            tag="div"
+          >
+            <template #service>
+              <span class="text-capitalize">{{
+                notification.i18nParam.props.service
+              }}</span>
+            </template>
+            <template #location>
+              {{ notification.i18nParam.props.location }}
+            </template>
+            <template #url>
+              <a :href="notification.i18nParam.props.url" target="_blank">{{
+                notification.i18nParam.props.url
+              }}</a>
+            </template>
+          </i18n>
```

Comment:
Btw wouldn't it be more clear semantically if we kept `message` only for text messages and used a key property in `i18n.param` for the key?

Reply:
yeah, you're right

Reply:
done